doc/tutorials/advanced/run-application-with-own-data.md

```diff
@@ -1,14 +1,18 @@
 # Run application with own data
```

Comment:
Are there any text changes in this file? Can you please point out where exactly are these changes? Thanks

Reply:
not sure, maybe lint removed some empty lines?